chromadb/test/property/test_filtering.py

```diff
@@ -206,7 +207,7 @@ def test_filterable_metadata_get(
         embedding_function=collection.embedding_function,
     )
 
-    initial_version = coll.get_model()["version"]
+    initial_version = cast(int, coll.get_model()["version"])
 
     coll.add(**record_set)
```

Comment:
Let's make it explicit that we cannot generate RecordSet without IDs here

Reply:
done